### PR TITLE
feat(1082): [3] add branchList function

### PR DESCRIPTION
**Pull request by yuichi10**
## Context
scm functions are called by this router. 

## Objective
Add branchList function.

## Reference
Related to screwdriver-cd/screwdriver#1082

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -357,6 +357,17 @@ class ScmRouter extends Scm {
...
     }
+
+    /**
+     * Get branch info of scmContext
+     * @method getBranchList
+     * @param  {Object}     config              Configuration
+     * @param  {String}     config.scmContext   Name of scm context
+     * @return {String}                         branch info of scmContext
+     */
+    getBranchList(config) {
```
Comment (tk3fftk) [inline index.js]: `_getBranchList`?

Comment (yuichi10) [reply index.js]: You are right _getBranchList is correct. I will fix.

**Review by tk3fftk — APPROVED** [review]
👍

**Review by s-yoshika — APPROVED** [review]

**Review by tkyi — COMMENTED** [review]
Probably should add a test

**Comment by yuichi10** [comment]
Ahhh I will add test!

**Comment by yuichi10** [comment]
@tkyi I add test!

**Review by tkyi — APPROVED** [review]
Thank you! 🙇‍♀️

**Pull request merged**